web/src/admin.rs

```diff
@@ -0,0 +1,91 @@
+use axum::{
+    extract::{FromRequestParts, State},
+    http::{request::Parts, StatusCode},
+    response::IntoResponse,
+    routing::get,
+};
+use cja::app_state::AppState as _;
+use color_eyre::eyre::Context;
+use db::users::User;
+
+use crate::{err, templates::Template, users::ExtractUserError, AppState, WebResult};
+
+async fn simple_error(_: AdminUser) -> WebResult<()> {
+    Err(color_eyre::eyre::eyre!("This is a test error"))?
+}
+
+async fn sql_error(_: AdminUser, State(app_state): State<AppState>) -> WebResult<()> {
+    let _ = sqlx::query("This is not valid sql")
+        .fetch_one(app_state.db())
+        .await
+        .wrap_err("We meant for this to fail but lets see the error chain")?;
+    Ok(())
+}
```

Comment:
These are two routes that are error on purpose to test error reporting and things like that!

I want to see these show up in Sentry.io